src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/showConfirmationPaymentFromComponent/payment.js

```diff
@@ -78,31 +78,39 @@ function handleAuthorisedPayment(
   return next();
 }
 
-function handlePayment(stateData, order, options) {
+function handlePayment(stateData, order, result = null, options) { // eslint-disable-line
```

Comment:
Why do we need to disable eslint? Is there a way to resolve this without doing that?